libvast/src/system/configuration.cpp

```diff
@@ -55,8 +56,14 @@ void initialize_factories() {
 configuration::configuration() {
   detail::add_message_types(*this);
   detail::add_error_categories(*this);
-  // Use 'vast.ini' instead of generic 'caf-application.ini'.
-  config_file_path = "vast.ini";
+  // Use 'vast.conf' instead of generic 'caf-application.ini' and fall back to
+  // $PREFIX/etc/vast if no local vast.conf exists.
+  if ((path::current() / "vast.conf").is_regular_file()) {
+    config_file_path = "vast.conf";
+  } else {
+    auto global_conf = path{VAST_INSTALL_PREFIX} / "etc/vast";
+    config_file_path = global_conf.str();
+  }
```

Comment:
I recall from our discussion that we also wanted to search `/etc` when the install prefix is different from `/`.